bigquery/integration_test.go

```diff
@@ -2016,18 +2016,16 @@ func TestIntegration_QueryStatistics(t *testing.T) {
 		t.Error("expected query timeline, none present")
 	}
 
-	if qStats.BIEngineStatistics == nil {
-		t.Error("expected BIEngine statistics, none present")
-	}
-
-	expectedMode := true
-	for _, m := range []string{"FULL", "PARTIAL", "DISABLED"} {
-		if qStats.BIEngineStatistics.BIEngineMode == m {
-			expectedMode = true
+	if qStats.BIEngineStatistics != nil {
+		expectedMode := true
```

Comment:
This value appears to always be true. Should it be starting as false?

Reply:
Doh.  Addressed.